site/src/atomics/spacing.md

```diff
@@ -6,7 +6,70 @@ template: standard
 
 # Spacing Atomics
 
+Spacing atomics can be used to add margin or padding on elements. They are especially helpful when composing layouts.
+
+| cssproperty                                                                                                                                                                                                            | value                                                   | screensize                        |
+| ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------- | --------------------------------- |
+| `margin`, `padding`, `margin-block`, `padding-block`,`margin-inline`, `padding-inline`, `margin-top`, `padding-top`, `margin-right`, `padding-right`, `margin-bottom`, `padding-bottom`, `margin-left`, `padding-left` | `xxs`, `xs`, `s`, `m`, `l`, `xl`, `xxl`, `xxxl`, `none` | `tablet`, `desktop`, `widescreen` |
+
+## Usage
+
+_Adding uniform spacing_
+
+Use `margin` or `padding` with a size value to add spacing around an element.
+
+```html
+<div class="margin-xxs border">XXS</div>
+<div class="margin-xs border">XS</div>
+<div class="margin-s border">X</div>
+<div class="margin-m border">M</div>
+<div class="margin-l border">L</div>
+<div class="margin-xl border">XL</div>
+<div class="margin-xxl border">XXL</div>
+<div class="margin-xxxl border">XXXL</div>
+```
+
+_Adding horizontal and vertical spacing_
+
+Use `margin-inline` and `padding-inline` with a size value to apply both left and right spacing.
+
+```html
+<div class="margin-inline-xl border">Margin-inline</div>
+<div class="padding-inline-xl border">Padding-inline</div>
+```
+
+Use `margin-block` and `padding-block` for top and bottom spacing.
+
+```html
+<div class="margin-block-xxl border">Margin-block</div>
+<div class="padding-block-xxl border">Padding-block</div>
+```
+
+_Adding spacing to a side_
+
+Use `margin-left` or `margin-right` with a size value to add left or right margin to an element.
```

Comment:
Remove italicized title?



Use `margin-<side>` or `padding-<side>` with a size value to add spacing to a particular side.

Reply:
Updated